backend/lib/endpoints/schema/organisations.js

```diff
@@ -82,5 +92,6 @@ module.exports = {
   deleteOrganisationSchema,
   getOrganisationSchema,
   getOrganisationsSchema,
```

Comment:
Great work @Zaydme working through this review - BE looks good moving to FE.

Not a blocker to approve, but I'm curious what's the rationale of adding to `getUsers` for search but creating a separate `searchOrganisations`?

Reply:
@joshmorel `getOrganisationsSchema` is already used to fetch the current user's orgnizations, so I had to add another `searchOrganisationsSchema`

Reply:
@Zaydme interestingly, I don't think `GET /organisations` is being used at all from the frontend (or at least from what I can tell using code search).  For current user's organisations we are including `organisations` virtual. 

But in any case what you have done is fine.